### PR TITLE
Changes for Bearer Token Support

### PR DESCRIPTION
**Pull request by merlynomsft**
**Task names**:

- AzureRmWebAppDeploymentV3
- AzureRmWebAppDeploymentV4
- AzureWebAppV1
- AzureFunctionAppContainerV1
- AzureWebAppContainerV1
- AzureAppServiceManageV0
- AzureFunctionAppV1
- AzureFunctionAppV2

**Description**: Applying changes from #18149 to support token auth in Kudu client.

**Documentation changes required:** N

**Added unit tests:** N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/AzureWebAppContainerV1/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "@types/node": "^10.17.0",
     "@types/q": "1.0.7",
     "@types/uuid": "^8.3.0",
-    "azure-pipelines-tasks-azurermdeploycommon": "3.208.0",
+    "azure-pipelines-tasks-azurermdeploycommon": "3.219.1",
```
Comment (merlynomsft) [inline Tasks/AzureWebAppContainerV1/package.json:24]: Publish azure-pipelines-tasks-azurermdeploycommon and update to 3.221.0

File: Tasks/AzureFunctionAppContainerV1/package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "@types/mocha": "^5.2.7",
     "@types/node": "^10.17.0",
     "@types/q": "1.0.7",
-    "azure-pipelines-tasks-azurermdeploycommon": "3.207.0",
+    "azure-pipelines-tasks-azurermdeploycommon": "3.219.1",
```
Comment (merlynomsft) [inline Tasks/AzureFunctionAppContainerV1/package.json:23]: Publish azure-pipelines-tasks-azurermdeploycommon and update to 3.221.0

File: Tasks/AzureAppServiceManageV0/package.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
     "@types/mocha": "^5.2.7",
     "@types/node": "^10.17.0",
     "@types/q": "1.0.7",
-    "azure-pipelines-tasks-azure-arm-rest-v2": "^3.218.0",
+    "azure-pipelines-tasks-azure-arm-rest-v2": "^3.219.1",
```
Comment (merlynomsft) [inline Tasks/AzureAppServiceManageV0/package.json:23]: Publish azure-pipelines-tasks-azure-arm-rest-v2 and update to 3.221.0

File: Tasks/AzureRmWebAppDeploymentV4/package.json [hunk 4]
```diff
@@ -20,7 +20,7 @@
     "@types/mocha": "^5.2.7",
     "@types/node": "^10.17.0",
     "@types/q": "1.0.7",
-    "azure-pipelines-tasks-azure-arm-rest-v2": "^3.217.1",
+    "azure-pipelines-tasks-azure-arm-rest-v2": "^3.219.0",
```
Comment (merlynomsft) [inline Tasks/AzureRmWebAppDeploymentV4/package.json:23]: Publish azure-pipelines-tasks-azure-arm-rest-v2 and update to 3.221.0

File: Tasks/AzureRmWebAppDeploymentV3/package.json [hunk 5]
```diff
@@ -22,7 +22,7 @@
     "@types/q": "1.0.7",
     "archiver": "1.2.0",
     "azure-pipelines-task-lib": "^3.4.0",
-    "azure-pipelines-tasks-azure-arm-rest-v2": "^3.220.2",
+    "azure-pipelines-tasks-azure-arm-rest-v2": "3.220.2",
```
Comment (merlynomsft) [inline Tasks/AzureRmWebAppDeploymentV3/package.json:25]: Publish azure-pipelines-tasks-azure-arm-rest-v2 and update to 3.221.0

File: Tasks/AzureWebAppV1/package.json [hunk 6]
```diff
@@ -20,7 +20,7 @@
     "@types/mocha": "^5.2.7",
     "@types/node": "^10.17.0",
     "@types/q": "1.0.7",
-    "azure-pipelines-tasks-azurermdeploycommon": "^3.208.0",
+    "azure-pipelines-tasks-azurermdeploycommon": "3.219.1",
```
Comment (merlynomsft) [inline Tasks/AzureWebAppV1/package.json:23]: Publish azure-pipelines-tasks-azurermdeploycommon and update to 3.221.0

File: Tasks/AzureRmWebAppDeploymentV4/package.json (hunk 4 above)
Comment (onetocny) [reply Tasks/AzureRmWebAppDeploymentV4/package.json:23]: Done

File: Tasks/AzureRmWebAppDeploymentV3/package.json (hunk 5 above)
Comment (onetocny) [reply Tasks/AzureRmWebAppDeploymentV3/package.json:25]: Done

File: Tasks/AzureAppServiceManageV0/package.json (hunk 3 above)
Comment (onetocny) [reply Tasks/AzureAppServiceManageV0/package.json:23]: Done

File: Tasks/AzureFunctionAppContainerV1/package.json (hunk 2 above)
Comment (onetocny) [reply Tasks/AzureFunctionAppContainerV1/package.json]: Done

File: Tasks/AzureWebAppContainerV1/package.json (hunk 1 above)
Comment (onetocny) [reply Tasks/AzureWebAppContainerV1/package.json]: Done

File: Tasks/AzureWebAppV1/package.json (hunk 6 above)
Comment (onetocny) [reply Tasks/AzureWebAppV1/package.json]: Done

**Review by manolerazvan — APPROVED** [review]

**Review by mmrazik — APPROVED** [review]

**Review by FinVamp1 — COMMENTED** [review]
Looks great, thank you.

**Pull request merged**